```
PS F:\usage\build> cmake --build .
MSBuild version 17.6.3+07e294721 for .NET Framework

1>Checking Build System
Building Custom Rule F:/usage/CMakeLists.txt
main.cpp

F:\test\vcpkg\installed\x64-windows\include\seasocks/WebSocket.h(33,10): 
fatal  error C1083: Cannot open include fil e: '../../../win32/win_unistd.h': No such file or directory [F:\usage\build\main.vcxproj] 
```
We need to install `win_unistd.h`, otherwise we cannot support `WIN32`.

https://github.com/mattgodbolt/seasocks/blob/7155ad2946ea454245051f3a915ea1bc1ff4cf6c/src/main/c/seasocks/WebSocket.h#L33
https://github.com/mattgodbolt/seasocks/blob/7155ad2946ea454245051f3a915ea1bc1ff4cf6c/src/main/c/CMakeLists.txt#L8

--- a/ports/seasocks/vcpkg.json
+++ b/ports/seasocks/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "Simple, small, C++ embeddable webserver with WebSockets support",
   "homepage": "https://github.com/mattgodbolt/seasocks",
   "license": "BSD-2-Clause",
-  "supports": "!osx & !android",
+  "supports": "!windows & !osx & !android",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/versions/s-/seasocks.json
+++ b/versions/s-/seasocks.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "3fccf932ff8423ef9e626f3f7e930a244bd1c990",
+      "git-tree": "2477e55b543e20d96c95546ff5a362d8a1a72044",
       "version": "1.4.5",
       "port-version": 0
     }

